test/extended/testdata/deployments/deployment-ignores-deployer.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: DeploymentConfig
+metadata:
+  annotations:
+    deploy.openshift.io/deployer-pod.ignore: "true"
+  name: database
+spec:
+  replicas: 1
+  selector:
+    name: database
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        name: database
+    spec:
+      containers:
+      - env:
+        - name: MYSQL_USER
+          value: user1MM
+        - name: MYSQL_PASSWORD
+          value: wvXqW5E0
+        - name: MYSQL_DATABASE
+          value: root
+        image: openshift/mysql-55-centos7:latest
```

Comment:
use openshift/origin here (it does not matter for this test, but we don't want somebody to copy&paste this in a test that will actually pull this.